domains/xosjs.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "jacksoncraft859",
+        "email": "pollardjackson@outlook.com"
+    },
+    "record": {
+        "CNAME": "acc468f2-4bc9-4dfa-8059-bc9edf2eb513.id.repl.co"
```

Comment:
Unable to access the CNAME.

Reply:
Again it’s in development and can not be deployed and on all of the time please know I can not yet afford to keep the server running 24/7